CachedEntry.php

```diff
@@ -28,10 +28,10 @@ public function __construct(string $userAgent, $clientHints, array $values)
         $clientHints = $clientHints ? ClientHints::factory($clientHints) : null;
         parent::__construct($userAgent, $clientHints);
 
-        $this->bot = $values['bot'] ?? '';
+        $this->bot = $values['bot'] ?? null;
         $this->brand = $values['brand'] ?? '';
-        $this->client = $values['client'] ?? '';
-        $this->device = $values['device'] ?? '';
+        $this->client = $values['client'] ?? null;
+        $this->device = $values['device'] ?? null;
         $this->model = $values['model'] ?? '';
         $this->os = $values['os'] ?? '';
```

Comment:
That one better should be null as well. 
```suggestion
        $this->os = $values['os'] ?? null;
```

Reply:
done :+1: